macros/stg_dedup_nested_id.sql

```diff
@@ -0,0 +1,35 @@
+{%- macro stg_dedup_nested_id(table, id, event_id, pk, sort_key) -%}
+{% set id = id %}
+WITH
+    added_columns AS (
+    SELECT
+        *,
+        (
+        SELECT
+            A.value
+        FROM
+          UNNEST(_ATTRIBUTES) AS a
```

Comment:
Bit picky, but please change to `_attributes` to be consistent with the rest of the code 🙂